resources/assets/less/bem/contest-list-item.less

```diff
@@ -94,6 +98,9 @@
     white-space: nowrap;
     @media @mobile {
       font-size: 14px;
+      text-overflow: initial;
+      overflow: initial;
+      white-space: initial;
```

Comment:
probably cleaner to move the overflow stuff into a `@media @desktop {` block instead of setting them on the base and then resetting them here